src/annotator/guest.ts

```diff
@@ -499,6 +496,14 @@ export class Guest extends TinyEmitter implements Annotator, Destroyable {
     }
   }
 
+  private async _scrollToAnnotation(tag: string) {
```

Comment:
Can you help me understand why this method is async?

Reply:
Ah, there is a mistake here! The last line of code should be `await this._scrollToAnchor(anchor)`. `_scrollToAnchor` is async because we allow the web app to intercept the action and cause scrolling to be deferred until the web app has been able to re-render.